css/style.css

```diff
@@ -1039,22 +1041,1370 @@ li { list-style-type: none; }
+    justify-content: space-between;
+    margin: 24px 33px 80.32px 15.33px;
+  }
+
+  .menu__logo {
+    display: flex;
+    width: 166px;
+    height: 51px;
+    background: url(https://i.ibb.co/x1r6ZBx/Tesla-Wordmark-Red.png;);
+    background-size: 100%;
+    background-position: left bottom 4px;
+    padding: 0;
+    margin: 0;
+  }
+
+  .menu__burger {
+    display: flex;
+    width: 35.23px;
+    height: 32px;
+    background: url(https://i.ibb.co/gzf1wRT/burger-ipad.png) no-repeat;
+    background-size: 100% 75%;
+    background-position: center center;
+    margin: 0 0 4px 0;
+    padding: 0;
+    border: 0;
+    outline: none;
+    cursor: pointer;
+  }
+
+  .container__sections {
+    display: flex;
+    flex-flow: column wrap;
+    max-width: 768px;
+    height: 100%;
+    justify-content: flex-start;
+    align-content: flex-start;
+  }
+
+  .sections__section {
+    display: flex;
+    flex-flow: column wrap;
+    max-width: 768px;
+    height: 100%;
+    justify-content: flex-start;
+    align-content: center;
+    background: url(https://i.ibb.co/YBt2P1f/Tesla-Wordmark-Red-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: top 90px center;
+    margin: 0;
+  }
+
+  .section__greeting {
+    display: flex;
+    width: 251px;
+    height: 19px;
+    font-size: 15px;
+    text-align: center;
+    margin: -14px auto 87px auto;
+    padding: 0;
+  }
+
+  .section__image-car {
+    display: flex;
+    width: 672.6px;
+    height: 327.64px;
+    padding: 0;
+    background: url(https://i.ibb.co/DWqCb1v/tesla-png-tesla-png-hd-1636-ipad.png);
+    background-size: contain;
+    margin-bottom: 0 0 36.55px 0;
+  }
+
+  .section__specifications {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: space-between;
+    justify-content: space-between;
+    width: 487.27px;
+    height: 196.14px;
+    margin: 0 auto 55px;
+    padding: 0;
+  }
+
+  .time-speed {
+    display: flex;
+    flex-flow: row wrap;
+    align-content: flex-start;
+    width: 225px;
+    height: 83.12px;
+    margin: 0;
+    padding: 0;
+  }
+
+  .time-speed__speedometer {
+    display: flex;
+    width: 65.92px;
+    height: 50px;
+    margin: 17.2px 0 0 0;
+    background: url(https://i.ibb.co/Pj85jGG/Speed-ipad.png) no-repeat;
+    background-size: 100%;
+    background-position: center bottom;
+  }
+
+  .time-speed__numeral {
+    display: flex;
+    width: 73px;
+    height: 62px;
+    margin: 1.43px 0 0 20.06px;
+    padding: 0;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 50px;
+    line-height: 61px;
+  }
+
+  .time-speed__text {
+    display: flex;
+    align-items: flex-start;
+    font-size: 20px;
+    line-height: 24px;
+    font-style: normal;
+    font-weight: 600;
+    text-align: center;
+    width: 53.03px;
+    height: 35.83px;
+    margin: 22px 0 0 5px;
+    padding: 0;
+  }
+
+  .subtext {
+    font-size: 12px;
+    line-height: 15px;
+    font-style: normal;
+    font-weight: normal;
+  }
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Hi-Tesla/pullRequest?prid=5851407)